releasenotes/notes/prelude-release-7.45.0-5ae9e29aa1cc832e.yaml

```
@@ -0,0 +1,5 @@
+prelude:
+    |
+    Release on: 2023-05-31
+
+    - Please refer to the `7.45.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7450>`_ for the list of changes on the Core Checks
```

this formatting looks really weird
```suggestion
    - Please refer to the [`7.45.0` tag on integrations-core](https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7450) for the list of changes on the Core Checks
```

---

This is automatically generated part. The marked part is translated to the hyperlink in browser. We were always doing it this way and not touching it, so I think it should stay the same.